### PR TITLE
Feature/decorator type

### PR DESCRIPTION
**Pull request by max-winderbaum**
This PR should resolve https://github.com/draft-js-plugins/draft-js-plugins/issues/476

### CODE REVIEW

**Comment by max-winderbaum** [comment]
Ah, looks like I'm loading Draft twice - I'm taking a look now.

**Comment by max-winderbaum** [comment]
I ended up pulling in ~85 lines of [Draft-Js Multidecorators](https://github.com/SamyPesse/draft-js-multidecorators) because it hasn't seen a commit since June 10 and it seems pretty stable - @nikgraf please let me know if you'd rather me have MultiDecorators as a dependency to the editor :)

File: draft-js-plugins-editor/src/Editor/MultiDecorator.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import Immutable from 'immutable';
+
+const KEY_SEPARATOR = '-';
+
+function MultiDecorator(decorators) {
```
Comment (max-winderbaum) [inline draft-js-plugins-editor/src/Editor/MultiDecorator.js]: Totally - on it now. I'll add the CHANGELOG.md as well :+1:

**Comment by nikgraf** [comment]
Alright, let's do it 👍

**Pull request merged**